config/migrationsConfig.js

```diff
@@ -3,7 +3,8 @@
 module.exports = {
     development: {
         url: process.env.DEV_DATASTORE_SEQUELIZE_URL,
-        logging: true
+        logging: true,
+        dialect: 'mysql'
```

Comment:
Should we specify dialect here? If we hardcode to mysql, would there be another file for postgres?

Reply:
Didn't intend to check-in this file. Fixed it.